### PR TITLE
[MM-24527] Add GenController documentation

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR adds documentation for `GenController`.

#### Ticket

https://mattermost.atlassian.net/browse/MM-24527

### CODE REVIEW

File: docs/generating_data.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Configuration
+
+First of all, the type of `UserController` to be used should be set to
+`generative` as explained [here](loadtest_config.md#usercontrollerconfiguration).
+
+### Copy and modify needed configuration file
+
+```sh
+cp config/gencontroller.default.json config/gencontroller.json
```
Comment (agnivade) [inline docs/generating_data.md:20]: I think this will change to sample.json after @isacikgoz's change.

File: docs/gencontroller_config.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+*float64*
+
+The percentage of direct channels to be created.
+
+## PercentGroupChannels
+
+*float64*
+
+The percentage of group channels to be created.
```
Comment (agnivade) [inline docs/gencontroller_config.md:55]: We should also mention that the sum of these should be equal to 1.

Comment (streamer45) [reply docs/gencontroller_config.md:55]: Good point. I'll add a note.

File: docs/generating_data.md (hunk 1 above)
Comment (streamer45) [reply docs/generating_data.md:20]: Yes, I think we can replace everything in one go when we merge #257.

**Pull request merged**